src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -62,4 +66,54 @@ public Entity getEntityFromDatastore(String entityName, String entityPropertyTit
     Entity entity = resultsList.get(0);
     return entity;
   }
+
+  /**
+  * Adds a game id to a user's list of games
+  */
+  public static boolean addGameToUser(Entity userEntity, DatastoreService datastore, String gameId) {
+ 
+    if(gameId == ""){
+        log.severe("no gameId");
+        return false;
+    }
+
+    if(userEntity == null || datastore == null){
+        log.severe("null datastore or user entity");
+        return false;
+    }
+ 
+    ArrayList<String> games = (ArrayList<String>) userEntity.getProperty("games");
+ 
+    if(games == null){
+        log.severe("no game list");
```

Comment:
Here we would want to log the username as well just so we know who we're talking about. However, if a user having no games is an expected failure mode, we might not want to log at all.

Reply:
Okay, I took this log out.